scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/UserIdentification.java

```diff
@@ -91,6 +91,11 @@ public abstract class UserIdentification implements Serializable, Comparable<Use
      */
     public abstract boolean isAllTenantPermission();
 
+    /**
+     *
```

Comment:
Any reason for this empty description?

Reply:
I must have forgotten to add it. Thanks.